### DISCUSSION TITLE
购买了一年版的专业版，当时能用了，第二天又变回了免费版。

### DISCUSSION THREAD

**Discussion by langmanbiji** [getlantern/lantern-forum-cn#767] [General]
我的邮箱是www_642714596@qq.com，邮箱内并没有抽到激活码。求官方解决
![微信图片_20240808192430](https://github.com/user-attachments/assets/55f2d22e-b51f-4a6c-8b92-d3dc1a42e274)

**Comment by MaxMKMax** [comment]
@langmanbiji 您好. 谢谢您提供的信息. 我会帮您提交问题的. 谢谢. 请您过几天留意这个帖子或您的邮箱.

**Reply by langmanbiji** [reply]
等的很着急，希望快点处理。我的邮箱www_642714596@qq.com

**Comment by langmanbiji** [comment]
过了这么长的时间了，为什么还没有结果。

**Reply by txwgoogol** [reply]
我续了两年，早知道这样就不续了😓
